### PR TITLE
fix: name and key tests

### PR DESCRIPTION
**Pull request by hacdias**

### CODE REVIEW

File: test/name.spec.js [hunk 1]
```diff
@@ -102,9 +100,7 @@ describe('.name', function () {
       return ipfs.name.resolve(name.Name)
         .then((res) => {
           expect(res).to.exist()
-          expect(res).to.be.eql({
-            Path: name.Value
-          })
+          expect(res).to.be.eql(name.Value)
```
Comment (diasdavid) [inline test/name.spec.js]: @hacdias should it be `name.value`?

Comment (hacdias) [reply test/name.spec.js]: Thanks for noticing 😄

**Comment by hacdias** [comment]
I think it's done @diasdavid. For some reason, some testes (not related to name nor key api) failed on some builds but not on others.

**Comment by diasdavid** [comment]
Let's fix the non related issues on a separate PR

**Pull request merged**